README.md

```diff
@@ -63,3 +63,27 @@ See how to use plugins in plugin documentation.
 #### C/C++
 
 - [dogebuild-c](https://github.com/dogebuild/dogebuild-c) - a C/C++ build plugin
+
+#### Go lang
+
+- [dogebuild-go](https://github.com/dogebuild/dogebuild-go) - a Go lang build plugin
+
+#### Free Pascal
+
+- [dogebuild-fpc](https://github.com/dogebuild/dogebuild-fpc) - a Free Pascal build plugin
```

Comment:
FPC plugin not adapted to current dogebuild state. I should archive it or adapt. However adaptation should be simple.

Reply:
done